Core/include/Acts/Visualization/EventDataView3D.hpp

```diff
@@ -208,6 +208,9 @@ struct EventDataView3D {
       const ViewConfig& smoothedConfig = s_viewSmoothed) {
     // @TODO: Refactor based on Track class
 
+    // TODO get particle hypothesis from MTJ
```

Comment:
From track I guess? Will you add this?

Reply:
there is not track accessible at the moment and I thought I should maybe keep this out of the PR. but I you think it makes sense I can refactor this method for tracks